### PR TITLE
Include bug fixes from the last release

### PR DESCRIPTION
**Pull request by sashimi36**

### CODE REVIEW

File: features/liquidity/components/BondLiquidityDialog.tsx [hunk 1]
```diff
@@ -56,17 +55,15 @@ export const BondLiquidityDialog = ({
...
-    tokenAmountInMicroDenom: maxLiquidityTokenAmount,
-  })
+  const [tokenAmount, setTokenAmount] = useState(0)
 
-  const [liquidityDollarAmount] = usePoolTokensDollarValue({
-    poolId,
-    tokenAmountInMicroDenom: tokenAmount,
-  })
+  const liquidityDollarAmount =
+    (tokenAmount / maxLiquidityTokenAmount) * maxDollarValueLiquidity
```
Comment (ben2x4) [inline features/liquidity/components/BondLiquidityDialog.tsx]: what does maxLiquidityTokenAmount mean? Maybe change to totalLiquidityTokenAmount and totalLiquidityDollarValue? 

I think making names more clear will help us onboard more devs.

File: features/liquidity/components/BondLiquidityDialog.tsx [hunk 2]
```diff
@@ -174,10 +171,11 @@ export const BondLiquidityDialog = ({
   const isLoading = isRequestingToBond || isRequestingToUnbond
 
   const handleAction = () => {
+    const amountForStaking = Math.floor(tokenAmount)
```
Comment (ben2x4) [inline features/liquidity/components/BondLiquidityDialog.tsx]: would changed to flooredTokenAmount

File: features/liquidity/components/PoolCard.tsx [hunk 3]
```diff
@@ -45,6 +48,7 @@ export const PoolCard = ({
   providedTotalLiquidity,
   stakedLiquidity,
   fluidLiquidity,
```
Comment (ben2x4) [inline features/liquidity/components/PoolCard.tsx:50]: what is fluid liquidity?

Comment (ben2x4) [reply features/liquidity/components/PoolCard.tsx:50]: maybe should be available liquidity?

File: features/liquidity/components/PoolCard.tsx [hunk 4]
```diff
@@ -58,8 +62,8 @@ export const PoolCard = ({
     ? formatToCompactDollarValue(providedTotalLiquidity.dollarValue)
     : 0
 
-  const totalDollarValueLiquidityFormatted = dollarValueFormatterWithDecimals(
-    fluidLiquidity.total.dollarValue,
+  const totalDollarValueLiquidityFormatted = dollarValueFormatter(
+    parseInt(String(fluidLiquidity.total.dollarValue), 10),
```
Comment (ben2x4) [inline features/liquidity/components/PoolCard.tsx]: what does the 10 do?

File: features/liquidity/components/StakingSummary.tsx [hunk 5]
```diff
@@ -22,16 +18,19 @@ type StakingSummaryProps = {
   maxLiquidity: number
   liquidityAmount: number
   onChangeLiquidity: (liquidityAmount: number) => void
+  maxLiquidityInDollarValue: number
```
Comment (ben2x4) [inline features/liquidity/components/StakingSummary.tsx]: same max here, also not consistent with above where we use `maxLiquidityDollarValue` without the `in`

File: features/liquidity/hooks/usePoolTokensDollarValue.ts [hunk 6]
```diff
@@ -1,14 +1,25 @@
 import { useSwapInfo } from 'hooks/useSwapInfo'
 import { useTokenDollarValue } from 'hooks/useTokenDollarValue'
-import { useBaseTokenInfo } from 'hooks/useTokenInfo'
+import {
+  PoolEntityType,
```
Comment (ben2x4) [inline features/liquidity/hooks/usePoolTokensDollarValue.ts:4]: what is PoolEntityType?

**Review by ben2x4 — COMMENTED** [review]
couple questions here!

File: features/liquidity/components/PoolCard.tsx (hunk 4 above)
Comment (sashimi36) [reply features/liquidity/components/PoolCard.tsx]: Second argument here sets out the math system. 10 in this case is for decimal one. I'm using the function to get rid of the decimal part. Could've probable just used `Math.ceil`

File: features/liquidity/hooks/usePoolTokensDollarValue.ts (hunk 6 above)
Comment (sashimi36) [reply features/liquidity/hooks/usePoolTokensDollarValue.ts:4]: Is a type for entities we have in pool json

**Pull request merged**